config/src/config/api_config.rs

```diff
@@ -83,7 +85,8 @@ pub const DEFAULT_MAX_SUBMIT_TRANSACTION_BATCH_SIZE: usize = 10;
 pub const DEFAULT_MAX_PAGE_SIZE: u16 = 100;
 const DEFAULT_MAX_ACCOUNT_RESOURCES_PAGE_SIZE: u16 = 9999;
 const DEFAULT_MAX_ACCOUNT_MODULES_PAGE_SIZE: u16 = 9999;
-const DEFAULT_MAX_VIEW_GAS: u64 = 2_000_000; // We keep this value the same as the max number of gas allowed for one single transaction defined in aptos-gas.
+const DEFAULT_MAX_VIEW_GAS: u64 = 500; // The value should be smaller than the max number of gas allowed for one single transaction defined in aptos-gas.
+const DEFAULT_MAX_SIMULATION_GAS: u64 = 500; // The value should be smaller than the max number of gas allowed for one single transaction defined in aptos-gas.
```

Comment:
I'm not sure you can just change this.  Would this break any existing applications, as well as this would prevent using simulation as a way to estimate gas usage